modules/ROOT/pages/_partials/rn-known-issues-ref.adoc

```diff
@@ -7,5 +7,5 @@ This section lists known issues with  {PRODUCT} {PATCH_VERSION_NO}:
 
 // * You do not have the required privileges to add {walkthrough-name}s to a cluster. If you require this feature, contact Red Hat Support for a workaround.
 
-* Any {PRODUCT_SHORT} user can determine what {walkthrough-name}s are available in a cluster.
-In a future release, this ability will be restricted to the `customer-admin` user.
+* Any {PRODUCT_SHORT} user can now determine which {walkthrough-name}s are available in a cluster. 
+In future releases, this ability will be restricted to the *customer-admin* user.
```

Comment:
I think the style guide suggests not promising future features. I think this was also brought up during Susan Jay's talk on standardizing release notes.

Reply:
@joaedwar Interesting, you got a link to that style guide?

Reply:
This particular issue is so egregious, it justifies an exception.
Any user can overwrite the walkthrough config of a cluster

Reply:
I see! Sorry, for rocking the boat :)